static/src/javascripts/projects/common/modules/onward/tech-feedback.js

```diff
@@ -86,16 +93,30 @@ define([
         }, {});
     }
 
+    function summariseAbTests(testsAndVariants) {
+        var testAndVariantPairs = pairs(testsAndVariants);
+        if (testAndVariantPairs.length === 0) {
+            return 'No tests running';
+        } else {
+            return map(testAndVariantPairs, function (pair) {
+                return pair[0] + '=' + pair[1];
```

Comment:
not sure if using the output of `getRunningTestVariants` is more readable than the original `getParticipations` function; now you're dealing with tuples rather than an object.